### ISSUE TITLE
Use <button> for buttons in the group entry

### ISSUE THREAD

**Issue by barmac** [bpmn-io/properties-panel#45]
These two are semantically buttons:

![image](https://user-images.githubusercontent.com/28307541/120455312-f355e180-c394-11eb-8d6a-6fba6f1b905d.png)

![image](https://user-images.githubusercontent.com/28307541/120455369-fd77e000-c394-11eb-9ce4-43aa3feb92a3.png)

When we use `\<div>`, it's not accessible via keyboard.

_Originally posted by @barmac in https://github.com/bpmn-io/properties-panel-next/issues/37#issuecomment-852862920_
